.chloggen/mackjmr_fix-container-tags-resource-attributes.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
Should this be a breaking change?

Reply:
Technically, it was a bug that we are reading from span attributes, but the change in behaviour could be considered a breaking change if people are relying on container tags which we extracted from span attributes.

Same for the other change, where technically adding the mapped DD tags as span tags was a bug, as they should only be added in `_dd.tags.container` (which are searchable without the `@`, vs span tags searchable by `@`). Again, the change in behaviour could be considered a breaking change if people are relying on span tags for their container tags.

I can mark both as breaking changes for the behavioural change, but this is more of a fix IMO. 

@mx-psi what do you think, we had briefly talked about this ?

Reply:
Are we documenting the current behavior anywhere? I would lean towards calling this one a bug fix, since container tags seem clearly like a resource-level concept and not a span-level concept. I am on the fence about the second one

Reply:
> Are we documenting the current behavior anywhere?

No, it's not documented anywhere.

Reply:
IMO it's fine to leave them as bug fixes if @songy23 is okay with it

Reply:
OK, I'm good with it too